src/pharmpy/tools/ruvsearch/tool.py

```diff
@@ -334,9 +342,16 @@ def _create_combined_model(input_model, current_iteration):
     return model
 
 
-def _create_dataset(input_model):
+def _create_dataset(input_model, dv):
     # Non-observations have already been filtered
     residuals = input_model.modelfit_results.residuals
+    if dv:
+        input_dataset = input_model.dataset
+        input_dataset = input_dataset[input_dataset['DV'] != 0].reset_index(
```

Comment:
Can I use "DV" here or can this also be a different column name?

Reply:
You could check the `datainfo` of the input model. It should have the `dv_column` property (which in turn has the `name` property)